### PR TITLE
Restart app between tests

### PR DESCRIPTION
**Pull request by maciej-szlosarczyk**
The `opentelemetry` app unfortunately is full of global state, so between each test case we need to check reset it's state.

### CODE REVIEW

**Comment by sauron-droid** [comment]
Thanks for your pull request. Before we can look at it, you'll need to add a 'DCO signoff' to your commits.

:memo: **Please follow instructions in the [contributing guide](https://github.com/mindvalley/mv-opentelemetry/blob/master/CONTRIBUTING.md) to update your commits with the DCO**

Full details of the Developer Certificate of Origin can be found at [developercertificate.org](https://developercertificate.org/).

**The list of commits missing DCO signoff**:

- [91311a0](https://github.com/mindvalley/mv-opentelemetry/commits/91311a0f00d8159bdc832b5419a96fdc7be29dbe) Restart app between tests

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
**Once this PR has been reviewed and has the lgtm label**, please ask for approval from maciej-szlosarczyk by writing `/assign @maciej-szlosarczyk` in a comment. For more information see [the Kubernetes Code Review Process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process).

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fmv-opentelemetry).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/mv-opentelemetry/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by antonsatin — DISMISSED** [review]

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/mv-opentelemetry/pull/41#pullrequestreview-1227364565" title="Approved">antonsatin</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fmv-opentelemetry).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/mindvalley/mv-opentelemetry/blob/main/OWNERS)~~ [antonsatin]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by sauron-droid** [comment]
LGTM label has been added.  <details>Git tree hash: ad2085cf77438fd1e5e5ed534b6c265719dfcf61</details>

**Comment by sauron-droid** [comment]
New changes are detected. LGTM label has been removed.

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
**Once this PR has been reviewed and has the lgtm label**, please ask for approval from antonsatin by writing `/assign @antonsatin` in a comment. For more information see [the Kubernetes Code Review Process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process).

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fmv-opentelemetry).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/mv-opentelemetry/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
**Once this PR has been reviewed and has the lgtm label**, please ask for approval from antonsatin by writing `/assign @antonsatin` in a comment. For more information see [the Kubernetes Code Review Process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process).

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fmv-opentelemetry).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/mv-opentelemetry/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by antonsatin — APPROVED** [review]

**Comment by sauron-droid** [comment]
LGTM label has been added.  <details>Git tree hash: 0247c6ed6dc1815228fb69ce6678379b7951a037</details>

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/mv-opentelemetry/pull/41#pullrequestreview-1227371577" title="Approved">antonsatin</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fmv-opentelemetry).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/mindvalley/mv-opentelemetry/blob/main/OWNERS)~~ [antonsatin]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
